### DISCUSSION TITLE
Suspicious activity blocked

### DISCUSSION THREAD

**Discussion by subdsys** [prasathmani/tinyfilemanager#1156] [General]
Suspicious activity blocked

Feature:
Antivirus

C:\Users\subdomain\Downloads\tinyfilemanager-master.zip tried to load a malicious resource detected as Trojan.Generic.35547372 and was blocked. Your device is safe

Am I missing something here? This attempted to install a Trojan on my box, is this expected?

**Comment by SamSirry** [comment]
Yes it is expected. This is a script that allows some people to access local files remotely. For the unaware person it's worth a warning.
